src/modules/security/index.ts

```diff
@@ -0,0 +1,145 @@
+import type { Faker } from '../..';
+import { toDate } from '../../internal/toDate';
+
+/**
+ * The possible definitions related to
+ * Common Vulnerability Scoring System (CVSS).
+ */
+export interface Cvss {
+  /**
+   * A value ranging from 0 to 10.
+   */
+  score: number;
+
+  /**
+   * A compressed textual representation of the values used to derive a score
+   */
+  vector: string;
+
+  /**
+   * A textual representation of the numeric score.
+   *
+   * Where:
+   * - None – 0
+   * - Low – 0.1 - 3.9
+   * - Medium – 4.0 - 6.9
+   * - High – 7.0 - 8.9
+   * - Critical – 9.0 - 10.0
+   */
+  rating: SeverityRating;
+}
+
+/**
+ * Possible textual rating definitions for a CVSS identifier
+ */
+type SeverityRating = 'none' | 'low' | 'medium' | 'high' | 'critical';
+
+/**
+ * Module to generate security related entries.
+ */
+export class Security {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(Security.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Generates a random CVE between the given boundaries.
+   * Based on:
+   * https://www.cve.org/
+   *
+   * @param options The options to use. Defaults to `{}`.
+   * @param options.from The early date boundary. Defaults to `1999-01-01T00:00:00.000Z`.
+   * @param options.to The late date boundary. Defaults to `now`.
+   *
+   * @example
+   * faker.security.cve() // 'CVE-2011-0762'
+   * faker.security.cve({ from:'2020-01-01T00:00:00.000Z', to: '2030-01-01T00:00:00.000Z' }) // 'CVE-2028-0762'
+   */
```

Comment:
```suggestion
   * @example
   * faker.security.cve() // 'CVE-2011-0762'
   * faker.security.cve({ from:'2020-01-01T00:00:00.000Z', to: '2030-01-01T00:00:00.000Z' }) // 'CVE-2028-0762'
   */
```
Could you please add two more example function calls, one with only the `from` option and the other one with only the `to` option?